src/HashingLogic.ts

```diff
@@ -204,9 +427,7 @@ export enum ChainId {
 }
 
 export const getAttestationAgreementEIP712 = (
```

Comment:
Did the EIP change? Why remove the `chainId` and `verifyingContract` params?

Reply:
They're optional and I didn't think they were super relevant for us